app/views/layouts/application.html.erb

```diff
@@ -1,118 +1,85 @@
+                </ul>
+              </li>
             </ul>
           </div>
         </div>
       </nav>
-    <% end %>
-    <% if flash[:notice] %>
+      <% if flash[:notice] %>
       <div class="alert alert-success">
-        <%= flash[:notice] %>
-      </div>
+      <%= flash[:notice] %>
+    </div>
     <% elsif flash[:alert] %>
-      <div class="alert alert-danger">
-        <%= flash[:alert] %>
-      </div>
-    <% end %>
+    <div class="alert alert-danger">
+    <%= flash[:alert] %>
+  </div>
+  <% end %>
 
-    <div class="container">
-      <%= yield %>
-    </div>
+  <div class="container">
+  <%= yield %>
+  </div>
 
-    <div class="bottomMenu navbar-fixed-bottom navbar-inverse">
-      <div class="navbar-left footer">
-        <ul class="nav navbar-nav">
-          <!-- <li><a href="#"></a></li> -->
-          <% if user_signed_in? %>
-            <li id="user-dropdown-menu" class="dropdown">
-              <a id="user-dropdown-button" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"><%= "Hello #{current_user.first_name}" %><span class="caret"></span></a>
-              <ul id="user-dropdown-items" class="dropdown-menu bottom-up" role="menu">
-                <!-- <li><a href="#">Action</a></li> -->
-                <li><%= link_to "Edit Profile", edit_user_path(current_user) %></li>
-                <li><%= link_to "Edit Password", edit_password_change_path(current_user) %></li>
-                <!-- <li class="divider"></li> -->
-                <!-- <li><a href="#">Separated link</a></li> -->
-                <li class="divider"></li>
-                <!-- <li><a href="#">One more separated link</a></li> -->
-                <li><%= link_to "Log out", sessions_path, method: :delete %></li>
-              </ul>
-            </li>
-          <% else %>
-            <!-- <li><a href="#">Sign In</a></li> -->
-            <li><%= link_to "Sign In", new_session_path %></li>
-            <li> <%= link_to "Sign Up", new_user_path, id: "btn-sign-up" %></li>
-          <% end %>
-        </ul>
-        <ul class="nav navbar-nav">
-          <li><%= link_to "About", about_path	 %></li>
-          <li><%= link_to "Create Organization", new_organization_path %></li>
-        </ul>
-      </div>
-      <div class="navbar-right footer-icons">
-        <i class="fa fa-twitter" aria-hidden="true"></i>
-        <i class="fa fa-facebook-official" aria-hidden="true"></i>
-        <i class="fa fa-google-plus" aria-hidden="true"></i>
-      </div>
-      <div class="navbar-right footer-text">
-          Built and maintained by students from <a href="http://codecore.ca">CodeCore</a>
-      </div>
-    </div>
+  <div class="bottomMenu navbar-fixed-bottom navbar-inverse">
+   <div class="navbar-left footer pull-left text-left">
+     <ul class="nav navbar-nav">
+       <li><%= link_to "About", about_path     %></li>
+       <li><%= link_to "Create Organization", new_organization_path %></li>
```

Comment:
I don't this the new organization button should be in the bottom navbar, people usually overlook data in the bottom navbar and this is the only way we have to create a new organization.